src/extension/background-script/actions/ln/invoices.ts

```diff
@@ -11,12 +11,15 @@ const invoices = async (message: MessageInvoices) => {
     const result = await connector.getInvoices();
     let invoices: Invoice[] = result.data.invoices
       .filter((invoice) => (isSettled ? invoice.settled : !invoice.settled))
-
-      .map((invoice: Invoice) => {
+      .map((invoice) => {
         const boostagram = utils.getBoostagramFromInvoiceCustomRecords(
           invoice.custom_records
         );
-        return { ...invoice, boostagram };
+        const type = invoice.type;
```

Comment:
undo this change, once type is reverted to "received" in `ConnectorInvoice` in connector.interface

Reply:
resolved